frameworks/hdfs/universe/config.json

```diff
@@ -1523,24 +1522,24 @@
           "default": true
         },
         "fs_s3n_block_size": {
-          "type": "integer",
+          "type": "string",
           "description": "Block size to use when reading files using the native S3 filesystem (s3n: URIs).",
-          "default": 67108864
+          "default": "67108864"
         },
         "fs_s3n_multipart_uploads_enabled": {
           "type": "boolean",
           "description": "Setting this property to true enables multiple uploads to native S3 filesystem. When uploading a file, it is split into blocks if the size is larger than fs.s3n.multipart.uploads.block.size.",
           "default": false
         },
         "fs_s3n_multipart_uploads_block_size": {
-          "type": "integer",
+          "type": "string",
           "description": "The block size for multipart uploads to native S3 filesystem. Default size is 64MB.",
-          "default": 67108864
+          "default": "67108864"
         },
         "fs_s3n_multipart_copy_block_size": {
-          "type": "integer",
+          "type": "string",
```

Comment:
Looking at https://developers.google.com/discovery/v1/type-format it seems that adding `"format": "int64"` here may be an option.

Reply:
That seemed reasonable, though I tried it and it didn't actually do anything (for eg, I made the format be `uint64` for a config field but the UI validation didn't actually complain when I put a negative number for the value so this extra attribute wouldn't actually be useful)